### ISSUE TITLE
Crashes when playing "Big Wave Small Wave" after "Revival Jam"

### ISSUE THREAD

**Issue by ttphi88** [adambennett/StS-DuelistMod#6]
Mod version: 2.476.0

Hi, the game crashes when I play "Big Wave Small Wave" after "Revival Jam". 
Please see the log below
```
02:50:09.139 INFO basemod.BaseMod> publish on post power apply
02:50:09.139 INFO basemod.BaseMod> powers modified
02:50:09.257 INFO basemod.BaseMod> publish on post power apply
02:50:09.257 INFO basemod.BaseMod> powers modified
02:50:09.373 INFO basemod.BaseMod> publish on post power apply
02:50:09.373 INFO basemod.BaseMod> powers modified
02:50:09.591 INFO basemod.BaseMod> publish start battle
02:50:09.591 INFO duelistmod.DuelistMod> random buff map turn num roll: 4
02:50:09.594 INFO duelistmod.DuelistMod> random buff map turn num roll: 4
02:50:09.596 INFO duelistmod.DuelistMod> random buff map turn num roll: 1
02:50:09.598 INFO animator.AnimatorPower> Applied PlayerStatistics
02:50:09.598 INFO animator.AnimatorPower> Clearing Player Stats
02:50:09.706 INFO basemod.BaseMod> publishPostDraw
02:50:09.707 INFO EYBModInitializer> Drawn: Swamp Frog+
02:50:09.823 INFO basemod.BaseMod> publishPostDraw
02:50:09.823 INFO EYBModInitializer> Drawn: Sparks
02:50:09.855 INFO basemod.BaseMod> publish on post power apply
02:50:09.855 INFO basemod.BaseMod> powers modified
02:50:10.590 INFO basemod.BaseMod> publishPostDraw
02:50:10.592 INFO EYBModInitializer> Drawn: Big Wave Small Wave
02:50:10.707 INFO basemod.BaseMod> publishPostDraw
02:50:10.708 INFO EYBModInitializer> Drawn: Aqua Spirit+
02:50:10.823 INFO basemod.BaseMod> publishPostDraw
02:50:10.824 INFO EYBModInitializer> Drawn: 7-Colored Fish
02:50:10.940 INFO basemod.BaseMod> publishPostDraw
02:50:10.941 INFO EYBModInitializer> Drawn: Harpie Feather Duster+
02:50:11.056 INFO basemod.BaseMod> publishPostDraw
02:50:11.057 INFO EYBModInitializer> Drawn: Revival Jam+
02:50:13.607 INFO basemod.BaseMod> publish on card use: theDuelist:HarpieFeather
02:50:13.607 INFO duelistmod.DuelistMod> Card Used: Harpie Feather Duster+
02:50:13.608 INFO duelistmod.DuelistMod> incremented spellsThisCombat, new value: 1
02:50:13.608 INFO duelistmod.DuelistMod> incremented spellRunCombat, new value: 17
02:50:13.608 INFO duelistmod.DuelistMod> added Harpie Feather Duster to spellsThisCombat, spellsThisRun
02:50:18.355 INFO basemod.BaseMod> publish on card use: theDuelist:SwampFrog
02:50:18.355 INFO duelistmod.DuelistMod> Card Used: Swamp Frog+
02:50:18.355 INFO duelistmod.DuelistMod> added Swamp Frog to monstersThisCombat, monstersThisRun
02:50:21.474 INFO basemod.BaseMod> publish on card use: theDuelist:RevivalJam
02:50:21.474 INFO duelistmod.DuelistMod> Card Used: Revival Jam+
02:50:21.474 INFO duelistmod.DuelistMod> added Revival Jam to monstersThisCombat, monstersThisRun
02:50:24.890 INFO basemod.BaseMod> publish on card use: theDuelist:BigWaveSmallWave
02:50:24.890 INFO duelistmod.DuelistMod> Card Used: Big Wave Small Wave
02:50:24.890 INFO duelistmod.DuelistMod> incremented spellsThisCombat, new value: 2
02:50:24.890 INFO duelistmod.DuelistMod> incremented spellRunCombat, new value: 18
02:50:24.890 INFO duelistmod.DuelistMod> added Big Wave Small Wave to spellsThisCombat, spellsThisRun
02:50:24.892 ERROR core.CardCrawlGame> Exception caught
java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at java.lang.String.substring(String.java:1967) ~[?:1.8.0_144]
	at duelistmod.powers.SummonPower.updateDescription(SummonPower.java:355) ~[DuelistMod.jar:?]
	at duelistmod.cards.BigWaveSmallWave.use(BigWaveSmallWave.java:72) ~[DuelistMod.jar:?]
	at com.megacrit.cardcrawl.characters.AbstractPlayer.useCard(AbstractPlayer.java:1659) ~[?:?]
	at com.megacrit.cardcrawl.actions.GameActionManager.getNextAction(GameActionManager.java:299) ~[?:?]
	at com.megacrit.cardcrawl.actions.GameActionManager.update(GameActionManager.java:151) ~[?:?]
	at com.megacrit.cardcrawl.rooms.AbstractRoom.update(AbstractRoom.java:326) ~[?:?]
	at com.megacrit.cardcrawl.dungeons.AbstractDungeon.update(AbstractDungeon.java:2494) ~[?:?]
	at com.megacrit.cardcrawl.core.CardCrawlGame.update(CardCrawlGame.java:871) ~[?:?]
	at com.megacrit.cardcrawl.core.CardCrawlGame.render(CardCrawlGame.java:422) [?:?]
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:225) [?:?]
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:126) [desktop-1.0.jar:?]
```

**Comment by adambennett** [comment]
This sucks, it happens consistently when you play those two cards together :/

Looking into fixes for this soon!

**Comment by adambennett** [comment]
Fixed with release of v2.483.0!
